src/contracts/BFactory.sol

```diff
@@ -33,10 +32,10 @@ contract BFactory is BBronze {
     _blabs = b;
   }
 
-  function collect(BPool pool) external {
+  function collect(address pool) external {
     require(msg.sender == _blabs, 'ERR_NOT_BLABS');
     uint256 collected = IERC20(pool).balanceOf(address(this));
-    bool xfer = pool.transfer(_blabs, collected);
+    bool xfer = IERC20(pool).transfer(_blabs, collected);
```

Comment:
We can update this after [BAL-77](https://linear.app/defi-wonderland/issue/BAL-77) and keep the parameter type as `IBPool` and this line as `bool xfer = pool.transfer(_blabs, collected);` since `IBPool` will inherit IERC20

Reply:
i'm ok with this change, tbh i don't like Interfaces arguments on the external methods, prefer raw ones.

Reply:
hmm, related to the following comments, i believe IBPool should inherit IERC20, so this change shouldn't be necessary